packages/template-retail-react-app/app/pages/product-list/partials/refinements.jsx

```diff
@@ -36,11 +36,11 @@ const Refinements = ({filters, toggleFilter, selectedFilters, isLoading}) => {
     // Getting the indices of filters to open accordions by default
     let filtersIndexes = filters?.map((filter, idx) => idx)
 
-    // Use saved state for accordions
-    if (!isServer) {
-        const savedExpandedAccordionIndexes =
-            window.localStorage.getItem(FILTER_ACCORDION_SATE) &&
-            JSON.parse(window.localStorage.getItem(FILTER_ACCORDION_SATE))
+    const [filterAccordionState, setFilterAccordionState] = useLocalStorage(FILTER_ACCORDION_SATE)
```

Comment:
When testing some changes with cookies disabled, this broke! Evidently it's the only place in the retail react app that directly uses local storage.